chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>Ye kafi upyogi hote hai koi decision(faisla) karne me. <s>Kyunki aap yaha tak pahuch chuke hai, aapne yaha tak pahuchne ki decision pehle hi le li hai !  </s></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>this is just to make readers think about daily life in terms of programming. This think this could be potentially though provoking. Wouldn't it be?</p>

Reply:
<p>It's not clear to me..  For me, examples like</p><p>if (sunny) do this</p><p>else if (rainy) do this</p><p>etc</p><p> </p><p>are more understandable.</p>